src/array.cr

```diff
@@ -1656,8 +1656,6 @@ class Array(T)
   # a.push("b", "c") # => ["a", "b", "c"]
   # ```
   def push(*values : T)
-    new_size = @size + values.size
-
     resize_if_cant_insert(values.size)
```

Comment:
sShouldn't `new_size` here be passed as an argument to `resize_if_cant_insert`?

Reply:
/cc @asterite as this is a leftover from #10081

Reply:
I think so